js-api/src/chem.ts

```diff
@@ -193,23 +193,25 @@ export namespace chem {
       return Sketcher.isEmptyMolfile(molFile);
     }
 
-    /** Sets the molecule, supports either SMILES or MOLBLOCK formats */
-    async setMolecule(molString: string, substructure: boolean = false) {
+    /** Sets the molecule, supports either SMILES, SNARTS or MOLBLOCK formats */
```

Comment:
What does the "substructure" parameter mean? Let's describe it in the documentation

Reply:
SNARTS